packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -229,13 +226,15 @@ class AllocationTableViewState extends State<AllocationTableView>
           controller.searchMatchMonitorAllocationsNotifier,
     );
 
+    final ScrollController scroller = ScrollController();
```

Comment:
nit: call `scrollController` not `scroller` for consistency with other code in DevTools.

Reply:
Done.